lib/livebook/live_markdown/import.ex

```diff
@@ -201,7 +201,9 @@ defmodule Livebook.LiveMarkdown.Import do
       | metadata: metadata,
         type: data["type"] |> String.to_existing_atom(),
         name: data["name"],
-        value: data["value"]
+        value: data["value"],
+        # Use default value for backward compatibility
+        reactive: Map.get(data, "reactive", false)
```

Comment:
I would actually take it further. I don't think we should persist it with a value of false, because there is no reason to do so and it helps keep the .livemd tidier.

Reply:
Good call!